src/Fantomas/CodePrinter.fs

```diff
@@ -3215,8 +3212,8 @@ and genApp astContext e es ctx =
                 (fun ctx ->
                     match es with
                     | [] -> false
-                    | [ h ]
-                    | h :: _ -> addSpaceBeforeParensInFunCall e h ctx)
+                    | [ h ] -> addSpaceBeforeParensInFunCall e h ctx
+                    | _ (* es.Length > 1 *)  -> true)
```

Comment:
Nitpick but please remove the comment `(* es.Length > 1 *)`, in the context of the pattern match it is pretty obvious that  `es` contains more than 1 element.

Reply:
Comment removed.